src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -14,7 +14,10 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public int matchResult(int points1, int points2, int bet1, int bet2) {
+        int result = (points1 < points2 && bet1 < bet2
```

Comment:
Good start, but try to think how you can make solution less complicated.
Hint:
you don't need separate variable "result"